src/views/dashboard/Calendar.vue

```diff
@@ -72,7 +72,6 @@ export default {
       this.fetchEvents()
     },
     eventClick (info) {
```

Comment:
this one in general is not required

Reply:
yes I was still working on showing the description of the events in some way, as they don't have a URL to redirect on click and give more information

Reply:
if there's a meet link we can use it I suppose, what do you think?

Reply:
yes or a link to the Facebook event, that is most often also in the description, but then you would have to write a function to parse the HTML data in the description and look for that specific word